study.md

```diff
@@ -50,15 +50,15 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+the controller answers requests it gets from the server then responds in json
```

Comment:
The router receives requests and decides how to handle them.  Controllers don't necessarily have to return JSON.